platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBeanTest.java

```diff
@@ -175,11 +184,53 @@ public void setUpTree() throws Exception {
     public void testInit() throws Exception {
         ApplicationServiceBean serviceBean = new ApplicationServiceBean(testAppService,
                 testConfigAdminExt);
+        serviceBean.setMBeanServer(mBeanServer);
         serviceBean.init();
 
-        //Do it again, should remove and re-init
+        verify(mBeanServer).registerMBean(any(Object.class), any(ObjectName.class));
+    }
+
+    /**
+     * Tests the {@link ApplicationServiceBean#init()} method for the case
+     * where the serviceBean has already been initialized
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testInitTwice() throws Exception {
+        ApplicationServiceBean serviceBean = new ApplicationServiceBean(testAppService,
+                testConfigAdminExt);
+        serviceBean.setMBeanServer(mBeanServer);
+        when(mBeanServer.registerMBean(any(Object.class), any(ObjectName.class)))
+                .thenThrow(new InstanceAlreadyExistsException()).thenAnswer(new Answer() {
```

Comment:
As far as I know, it should be throw an exception first, then return `null`, since the first `registerMBean` is the one that exists in the `try{}` block, so that's the once that would normally throw the `InstanceAlreadyExistsException` if it had already been registered. Then it's set to return `null` since the second `registerMBean` call should not be throwing an exception in this scenario.